### ISSUE TITLE
Exception in Exception Handler

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3157]
I've been seeing the following exception fairly often.  It arises while an exception is being handled:

```
During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "c:\Tom\git\leo-editor\leo\core\leoGlobals.py", line 304, in new_cmd_wrapper
    func(self, event=event)

  File "c:\Tom\git\leo-editor\leo\core\leoCommands.py", line 788, in executeScript
    g.handleScriptException(c, script_p, script, script1)

  File "c:\Tom\git\leo-editor\leo\core\leoGlobals.py", line 7032, in handleScriptException
    fileName, n = g.es_exception(full=full)

TypeError: cannot unpack non-iterable NoneType object
```

From the stack trace, it ought to be easy to fix.

**Comment by edreamleo** [comment]
@tbpassin Thanks for this. I thought I had removed the `full` kwarg everywhere.

**Comment by edreamleo** [comment]
Closed via PR #3157.
